packages/connect/e2e/__fixtures__/signTransactionMultisig.ts

```diff
@@ -75,7 +78,42 @@ export default {
         },
         // https://tbtc1.trezor.io/api/tx/4123415574c16899b4bb5b691f9b65643dbe566a9b68e4e2e7a8b29c79c83f2b
         {
-            description: 'Testnet (multisig): 2 of 3 (sign with 3rd key)',
+            description: 'Testnet (multisig): 2 of 3 (sign with 3rd key) LEXICOGRAPHIC',
```

Comment:
I would expect this test or the one below using `PRESERVED` to fail. But they are passing. Investigating...